.eslintrc.js

```diff
@@ -44,5 +35,53 @@ module.exports = {
         },
       },
     ],
+    'no-restricted-globals': [
+      'error',
+      {
+        name: '__dirname',
+        message: 'Not available in JavaScript',
+      },
+      {
+        name: '__filename',
+        message: 'Not available in JavaScript',
+      },
+      {
+        name: 'atob',
+        message:
+          "'atob' unavailable in React Native 0.72. Use 'decodeBase64' helper in src/obfuscation.ts instead",
+      },
+      {
+        name: 'btoa',
+        message:
+          "'btoa' unavailable in React Native 0.72. Use 'encodeBase64' helper in src/obfuscation.ts instead",
+      },
+      {
+        name: 'Buffer',
+        message:
+          "'Buffer' unavailable in JavaScript. Use 'Uint8Array' instead. For Base64, use helpers in src/obfuscation.ts",
+      },
+      {
+        name: 'clearImmediate',
+        message: "'clearImmediate' unavailable in JavaScript.",
+      },
+      {
+        name: 'process',
```

Comment:
👍  although looks like this is showing up in a `lint-test-sdk` error

Reply:
I can write extension specific overrides. One sec; I'll update.

Reply:
updated!